controls/radfiledialogs/dialog-controls/radopenfiledialog.md

```diff
@@ -54,9 +80,13 @@ The dialog supports single and multiple selection modes. By default you can sele
 
 #### __[C#] Example 3: Enable multiple selection__
 {{region cs-radfiledialogs-radopenfiledialog-2}}
-	RadOpenFileDialog openFileDialog = new RadOpenFileDialog();
-	openFileDialog.Owner = theHostWindowInstance;
-	openFileDialog.Multiselect = true;
+	private void ShowOpenFileDialog()

```

Comment:
Why this snippet is so compact, but the others are extended? I think we can make them consistent. Or we can have the first one extended and the others to be more compact.